Symfony/CS/Error/Error.php

```diff
@@ -46,28 +51,43 @@ class Error
     private $filePath;
 
     /**
-     * @param string $type
-     * @param string $filePath
+     * @var \Exception
+     */
+    private $source;
+
+    /**
+     * @param string     $type
+     * @param string     $filePath
+     * @param \Exception $source
      */
-    public function __construct($type, $filePath)
+    public function __construct($type, $filePath, \Exception $source = null)
```

Comment:
hmmm this will be in conflict with https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1285